code/__DEFINES/jobs.dm

```diff
@@ -44,6 +44,8 @@
 #define CLOWN			(1<<11)
 #define MIME			(1<<12)
 #define ASSISTANT		(1<<13)
+#define GIMMICK 		(1<<14)
+#define BARBER		    (1<<15)

```

Comment:
Gimmick and barber but no magician?

Reply:
I knew i forgot him somewhere. will fix.